ios/FluentUI/Vnext/Drawer/MSFSlideOutPanelElevation.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+
+// MARK: - CA Layer
+
+struct MSFSlideOutPanelElevation: ViewModifier {
+    var direction: MSFDrawerDirection
+    var tokens: MSFDrawerTokens
+
+    private struct LegacyShadowConstant {
+        static let shadowRadius: CGFloat = 4
+        static let shadowOffset: CGFloat = 2
+        static let shadowOpacity: Double = 0.05
+        static let shadowColor: Color = .black
+    }
+
+    func body(content: Content) -> some View {
+        switch direction {
+        case .left,
+             .right:
+            return AnyView(content
+                            .shadow(color: tokens.shadow1Color,
+                                    radius: tokens.shadow1Blur,
+                                    x: tokens.shadow1DepthX,
+                                    y: tokens.shadow1DepthY)
+                            .shadow(color: tokens.shadow2Color,
+                                    radius: tokens.shadow2Blur,
+                                    x: tokens.shadow2DepthX,
+                                    y: tokens.shadow2DepthY))
+        case.top:
+            return AnyView(
+                content
+                    .shadow(color: LegacyShadowConstant.shadowColor.opacity(LegacyShadowConstant.shadowOpacity),
+                            radius: LegacyShadowConstant.shadowRadius,
+                            x: 0,
+                            y: LegacyShadowConstant.shadowOffset)
+            )
+        case.bottom:
+            return AnyView(
+                content
+                    .shadow(color: LegacyShadowConstant.shadowColor.opacity(LegacyShadowConstant.shadowOpacity),
+                            radius: LegacyShadowConstant.shadowRadius,
+                            x: 0,
+                            y: -LegacyShadowConstant.shadowOffset)
+            )
+        }
+    }
+}
+
+struct RoundedCorner: Shape {
+
+    var radius: CGFloat = .infinity
+    var direction: MSFDrawerDirection = .left
+
+    var corners: UIRectCorner {
+        switch direction {
+        case .top:
+            return [.bottomLeft, .bottomRight]
+        case .bottom:
+            return [.topLeft, .topRight]
+        default:
+            return []
+        }
+    }
+    func path(in rect: CGRect) -> Path {
```

Comment:
Why can't we use the SwiftUI [cornerradius](https://developer.apple.com/documentation/swiftui/view/cornerradius(_:antialiased:)) modifier instead?
```

    /// Clips this view to its bounding frame, with the specified corner radius.
    ///
    /// By default, a view's bounding frame only affects its layout, so any
    /// content that extends beyond the edges of the frame remains visible. Use
    /// `cornerRadius(_:antialiased:)` to hide any content that extends beyond
    /// these edges while applying a corner radius.
    ///
    /// The following code applies a corner radius of 25 to a text view:
    ///
    ///     Text("Rounded Corners")
    ///         .frame(width: 175, height: 75)
    ///         .foregroundColor(Color.white)
    ///         .background(Color.black)
    ///         .cornerRadius(25)
    ///
    /// ![A screenshot of a rectangle with rounded corners bounding a text
    /// view.](SwiftUI-View-cornerRadius.png)
    ///
    /// - Parameter antialiased: A Boolean value that indicates whether the
    ///   rendering system applies smoothing to the edges of the clipping
    ///   rectangle.
    ///
    /// - Returns: A view that clips this view to its bounding frame with the
    ///   specified corner radius.
    @inlinable public func cornerRadius(_ radius: CGFloat, antialiased: Bool = true) -> some View
```

Reply:
`cornerRadius` clips in all direction. Below is the image with using corner Radius. 

![Screen Shot 2021-03-01 at 7 51 23 PM](https://user-images.githubusercontent.com/63682282/109594422-9c0b5100-7ac7-11eb-959e-bb2a6aa59213.png)

We want corner radius only in direction of presentation.